ILSANG/Sources/Views/Ranking/RankingItemView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  RankingItemView.swift
+//  ILSANG
+//
+//  Created by Kim Andrew on 10/7/24.
+//
+
+import SwiftUI
+
+struct RankingItemView: View {
+    
+    let level: Int
+    let rank: Rank
+    
+    var body: some View {
+        HStack (spacing: 17) {
+            RankingProfile(profileImage: nil, level: level)
+            
+            VStack (alignment: .leading, spacing: 3) {
+                Text(rank.nickname)
+                    .font(.system(size: 15))
+                    .fontWeight(.bold)
+                Text("\(rank.xpType): \(rank.xpPoint)p")
+                    .font(.system(size: 13))
+                    .foregroundColor(.gray400)
```

Comment:
```suggestion
                if let xpStat = XpStat(rawValue: rank.xpType.lowercased())?.headerText {
                    Text("\(xpStat) : \(rank.xpPoint)p")
                        .font(.system(size: 13))
                        .foregroundColor(.gray400)
                }
```

이전에 남겼던 리뷰 같긴한데, 한글로 보여야해서요 !
이렇게 수정하면 좋을 것 같습니다!!